pages/beta/admin/site/posts.vue

```diff
@@ -78,7 +78,14 @@
               </td>
               <td>{{ formatDate(post.created_at) }}</td>
               <td>{{ formatDate(post.last_modified) }}</td>
-              <td />
+              <td>
+                <NuxtLinkLocale
```

Comment:
Do we want to link the post title to the edit page too to make it work like the other pages ?

Reply:
I think we need to do a pass on this on all tables. Do we want to always link to the update page or to the public page and add a edit button in the last column? Need to check this with @agarrone